src/pybind/mgr/devicehealth/module.py

```diff
@@ -306,7 +347,7 @@ def put_device_metrics(self, ioctx, devid, data):
             log.exception("Error reading OMAP: {0}".format(e))
```

Comment:
log.exception("Error reading OMAP: {0}".format(e)) code exception.
Correct to 
self.log.exception(...)